src/alph_puzzle.c

```diff
@@ -859,9 +849,9 @@ static int AlphPuzzleMainSeq_RotateTile_impl(AlphPuzzleData *data) {
         data->subState++;
         break;
     case 1:
-        u16 temp = data->subStateTimer++;
+        u16 temp = data->sceneTImer++;
```

Comment:
data->sceneTImer -> data->sceneTimer (I missed the struct somewhere so update it there too)